### PR TITLE
[WIP] Get rid off get_details()

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=================

`get_details` should be replaced by getting the data directly from a view.

### CODE REVIEW

**Comment by quarckster** [comment]
@mshriver Please take a look

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -599,6 +599,28 @@ def read(self):
         return [{key: col.text for key, col in row} for row in self]
 
 
+class ParametrizedSummaryTable(ParametrizedView):
```
Comment (mshriver) [inline widgetastic_manageiq/__init__.py]: Good idea, I like it, reducing the boilerplate summary table widgets on most views.

There is a downside to this tradeoff, in that the view doesn't have an explicit implementation of the expected summary table titles.

I think its a worthy tradeoff, and that we can quickly and easily automate a test of what summary tables are available.

File: cfme/ansible/repositories.py [hunk 2]
```diff
@@ -40,12 +40,7 @@ class RepositoryDetailsView(RepositoryBaseView):
     refresh = Button(title="Refresh this page")
     configuration = Dropdown("Configuration")
     download = Button(title="Download summary in PDF format")
-
-    @View.nested
-    class entities(View):  # noqa
-        properties = SummaryTable("Properties")
-        relationships = SummaryTable("Relationships")
-        repository_options = SummaryTable("Repository Options")
+    entity = ParametrizedView.nested(ParametrizedSummaryTable)
```
Comment (mshriver) [inline cfme/ansible/repositories.py]: I think I would prefer a consistent implementation of the table widget structure, and the `summary` name.  `view.entities.summary('title')` is a good call structure, if we can keep consistency.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 53d3bc8474a2907ad7f85e2b07c5177cf0ec6852:

`cfme/infrastructure/virtual_machines.py`:
- :red_circle: Line 14:1: **F401** *'widgetastic_manageiq.CheckableManageIQTree' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**